.github/workflows/release_build.yml

```diff
@@ -31,6 +31,23 @@ jobs:
       lane: review
       slack_thread_ts: ${{needs.start_slack_thread.outputs.thread_ts}}
       version: ${{ github.ref_name }}
+
+  notify_ios_failure:
+    name: Notify ios Failure
+    needs: [start_slack_thread, ios]
+    if: ${{ failure() }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify Slack on iOS failure
+        env:
+          SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
+        run: |
+          curl -X POST \
+               -H 'Authorization: Bearer '"$SLACK_API_TOKEN" \
+               -H 'Content-type: application/json' \
+               -d '{"channel": "va-mobile-app", "text": "The release build failed for ios Version '${{ inputs.version }}', See <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|workflow run> for details.", "thread_ts": "${{ needs.start_slack_thread.outputs.thread_ts }}"}' \
```

Comment:
Some change requests:

1. `${{ inputs.version}}` won't work here since there is no such input. For the version, you'll have to use `${{ github.ref_name }}`. This was my fault as I copied this code for you from a different workflow
1. Please fix formatting errors: 
    1. Change casing for "ios" to be "iOS"
    1. "Version" shouldn't be capitalized
    1. Comma after the version should be a period